lobsterpy/cohp/analyze.py

```diff
@@ -433,7 +432,7 @@ def _get_orbital_resolved_data(
 
                 orb_bonding_dict_data = {}
                 # For each orbital collect the contributions of summed bonding
-                # and antibonding interactions seperately
+                # and antibonding interactions separately

```

Comment:
I will always need a spell checker for this one. Even in German ... Separat.

Reply:
Codespell checked it, wonder how it got left out